### PR TITLE
Drop logrus.Errorf to Warnf on warnings about OCI versus Docker format images.

### PR DESCRIPTION
**Pull request by rhatdan**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @vrothberg @TomSweeneyRedHat @saschagrunert @QiWang19 @ashley-cui 
Fixes: https://github.com/containers/buildah/issues/2611

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by QiWang19** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Did you sign this @rhatdan?  gatings is claiming not DCO.

File: docs/buildah.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The Buildah package provides a command line tool which can be used to:
 
 **--log-level** **value**
 
-The log level to be used. Either "debug", "info", "warn" or "error", per default "error".
+The log level to be used. Either "debug", "info", "warn" or "error", per default "warn".
```
Comment (TomSweeneyRedHat) [inline docs/buildah.md:25]:
```suggestion
The log level to be used. Either "debug", "info", "warn" or "error"; per default "warn".
```

**Comment by TomSweeneyRedHat** [comment]
one small nitty suggestion otherwise 
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Failures are fixed by @cevich PR once we get it in, we should be able to merge all of these PRs.

**Comment by nalind** [comment]
LGTM

**Comment by rhatdan** [comment]
@saschagrunert @giuseppe @vrothberg @TomSweeneyRedHat @nalind @ashley-cui Tests are green, can we get this merged.

**Review by giuseppe — APPROVED** [review]
/lgtm

**Pull request merged**